src/Popover.tsx

```diff
@@ -158,9 +158,9 @@ export const Popover = forwardRef<HTMLElement, PopoverProps>(
       return () => {
         Object.keys(containerStyle ?? {}).forEach(
           (key) =>
-            (popoverElement.style[
+            delete popoverElement.style[
```

Comment:
You cannot set the value of `CSSStyleDeclaration` to null. Instead you must just delete the key value pair.